### PR TITLE
CON-2206: cloudPress raid for afpa

### PR DESCRIPTION
**Pull request by MSilva95**

### CODE REVIEW

File: subjects/devops/cloudPress/audit/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+###### Does the README.md file contain an architecture overview?
+
+###### Does the documentation cover the deployment process comprehensively?
+
+###### Are additional configurations, tips, and potential pitfalls included in the documentation?
+
+#### Evaluation Criteria
+
+###### Was WordPress and MariaDB set up using Terraform and Ansible?
```
Comment (nprimo) [inline subjects/devops/cloudPress/audit/README.md:57]: This question looks like a repetition: on line 25 it is asked if MariaDB was installed using Ansible and on line 41 it is asked if WordPress was installed with Ansible. 

I feel this question could be removed

File: subjects/devops/cloudPress/audit/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+###### Was the environment configuration secure and effective?
```
Comment (nprimo) [inline subjects/devops/cloudPress/audit/README.md:59]: I would add some details about "secure and effective". For example:
- no password or sensible information were hardcoded in the configuration files 
- password and sensible information are handled using proper tools like AWS Secrets Manager, HashiCorp Vault or any equivalent
- ...

File: subjects/devops/cloudPress/README.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+- Architecture overview.
+- Deployment process.
+- Additional configurations made.
+- Tips and potential pitfalls.
+
+### Evaluation Criteria
+
+Your CloudPress project will be evaluated based on:
+
+- Successfully setting up WordPress and MariaDB using Terraform to create the infrastructure and Ansible for installing everything.
```
Comment (nprimo) [inline subjects/devops/cloudPress/README.md:78]: I would add **nginx** to the required things to set up:
` ... setting up WordPress, MariaDB and Nginx..`

File: subjects/devops/cloudPress/README.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
+## CloudPress
+
+![Cloud Deployment](./resources/architecture.png)
+
+### Objective
+
+CloudPress is a project designed to provide hands-on experience in deploying a WordPress website with MariaDB on the Amazon Web Services (AWS) cloud platform. The goal is to utilize Terraform for infrastructure creation and Ansible for application installation, resulting in a production-ready setup.
```
Comment (nprimo) [inline subjects/devops/cloudPress/README.md:7]: I would add Ngnix here as well
`... deploying a WordPress website with MariaDB and Nginx ...`

**Review by nprimo — COMMENTED** [review]
It looks good to me :) I have added a couple of suggestions in line.

File: subjects/devops/cloudPress/README.md (hunk 4 above)
Comment (MSilva95) [reply subjects/devops/cloudPress/README.md]: done

File: subjects/devops/cloudPress/README.md (hunk 3 above)
Comment (MSilva95) [reply subjects/devops/cloudPress/README.md]: done

File: subjects/devops/cloudPress/audit/README.md (hunk 1 above)
Comment (MSilva95) [reply subjects/devops/cloudPress/audit/README.md]: done

File: subjects/devops/cloudPress/audit/README.md (hunk 2 above)
Comment (MSilva95) [reply subjects/devops/cloudPress/audit/README.md]: done

**Review by MSilva95 — COMMENTED** [review]
i did id the changes and added some questions as well

File: subjects/devops/cloudPress/audit/README.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+###### Is PHP-FPM running? Use `systemctl status php-fpm`.
+
+###### Were PHP settings appropriately configured for the WordPress site?
+
+###### Was the PHP service successfully started?
+
+#### WordPress Installation and Configuration using Ansible
+
+###### Was WordPress downloaded and set up on the EC2 Instance using Ansible?
```
Comment (Frenchris) [inline subjects/devops/cloudPress/audit/README.md:47]: here maybe we can add the WPcommand `wp core version` guide the corrector?

File: subjects/devops/cloudPress/audit/README.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+#### Technical Verification
+
+##### Try to change Terraform Settings.
+
+Example:
+
+- Modify the count of EC2 instances. For instance, if you initially provisioned two instances, change the count to three or reduce it to one.
+- Alter inbound or outbound rules in the security groups, like opening or closing specific ports.
+- Others that you feel are relevant.
```
Comment (Frenchris) [inline subjects/devops/cloudPress/audit/README.md:89]: This sentence is missing a verb. like "change" or "alter"  at the beggining.

File: subjects/devops/cloudPress/audit/README.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+
+###### Does the README.md file contain an architecture overview?
+
+###### Does the documentation cover the deployment process comprehensively?
+
+###### Are additional configurations, tips, and potential pitfalls included in the documentation?
+
+#### Technical Verification
+
+##### Try to change Terraform Settings.
```
Comment (Frenchris) [inline subjects/devops/cloudPress/audit/README.md:83]: I would change specify the question a bit. “Change some Terraform settings as instructed below:”

**Review by Frenchris — APPROVED** [review]

**Pull request merged**